docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -124,10 +124,7 @@ To examine alerts associated with the event, select the alert pill (*_x_ alert*)
 
 In the example screenshot below, five alerts were generated by the analyzed event (`lsass.exe`). The left pane displays the associated alerts and basic information about each one.
 
-preview::[]
-
-NOTE: Displaying alerts in the process tree is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature. In addition, to display it in {elastic-sec} you must add the following feature flag to the `kibana.yml` file:
-`xpack.securitySolution.enableExperimental: ['insightsRelatedAlertsByProcessAncestry']`
+NOTE: Displaying alerts in the process tree is a https://www.elastic.co/pricing[Platinum or Enterprise subscription] feature.
```

Comment:
This might be a bit out of scope, but it'd be nice if we could mention the licensing when we _first_ start describing this feature, instead of taking two paragraphs to describe it and _then_ saying "by the way, all that requires a subscription."

Maybe start 2 paragraphs above with something like "If you have a Platinum or Enterprise subscription, you can also examine alerts associated with events..." or whatever the subscription actually enables.